simde/x86/svml.h

```diff
@@ -4768,13 +4722,213 @@ simde_mm512_mask_log_pd(simde__m512d src, simde__mmask8 k, simde__m512d a) {
   #define _mm512_mask_log_pd(src, k, a) simde_mm512_mask_log_pd(src, k, a)
 #endif
 
+SIMDE_FUNCTION_ATTRIBUTES
+simde__m128
+simde_mm_cdfnorminv_ps (simde__m128 a) {
+  #if defined(SIMDE_X86_SVML_NATIVE) && defined(SIMDE_X86_SSE_NATIVE)
+    return _mm_cdfnorminv_ps(a);
+  #elif (SIMDE_NATURAL_VECTOR_SIZE > 0)
+    const simde__m128 a_[] = { simde_mm_set1_ps(SIMDE_FLOAT32_C(-3.969683028665376e+01)),
+                              simde_mm_set1_ps(SIMDE_FLOAT32_C( 2.209460984245205e+02)),
+                              simde_mm_set1_ps(SIMDE_FLOAT32_C(-2.759285104469687e+02)),
+                              simde_mm_set1_ps(SIMDE_FLOAT32_C( 1.383577518672690e+02)),
+                              simde_mm_set1_ps(SIMDE_FLOAT32_C(-3.066479806614716e+01)),
+                              simde_mm_set1_ps(SIMDE_FLOAT32_C( 2.506628277459239e+00))
+                            };
+
+    const simde__m128 b[] = { simde_mm_set1_ps(SIMDE_FLOAT32_C(-5.447609879822406e+01)),
+                              simde_mm_set1_ps(SIMDE_FLOAT32_C( 1.615858368580409e+02)),
+                              simde_mm_set1_ps(SIMDE_FLOAT32_C(-1.556989798598866e+02)),
+                              simde_mm_set1_ps(SIMDE_FLOAT32_C( 6.680131188771972e+01)),
+                              simde_mm_set1_ps(SIMDE_FLOAT32_C(-1.328068155288572e+01)),
+                            };
+
+    const simde__m128 c[] = { simde_mm_set1_ps(SIMDE_FLOAT32_C(-7.784894002430293e-03)),
+                              simde_mm_set1_ps(SIMDE_FLOAT32_C(-3.223964580411365e-01)),
+                              simde_mm_set1_ps(SIMDE_FLOAT32_C(-2.400758277161838e+00)),
+                              simde_mm_set1_ps(SIMDE_FLOAT32_C(-2.549732539343734e+00)),
+                              simde_mm_set1_ps(SIMDE_FLOAT32_C( 4.374664141464968e+00)),
+                              simde_mm_set1_ps(SIMDE_FLOAT32_C( 2.938163982698783e+00))
+                            };
+
+    const simde__m128 d[] = { simde_mm_set1_ps(SIMDE_FLOAT32_C( 7.784695709041462e-03)),
+                              simde_mm_set1_ps(SIMDE_FLOAT32_C( 3.224671290700398e-01)),
+                              simde_mm_set1_ps(SIMDE_FLOAT32_C( 2.445134137142996e+00)),
+                              simde_mm_set1_ps(SIMDE_FLOAT32_C( 3.754408661907416e+00)),
+                            };
+
+    const simde__m128 low  = simde_mm_set1_ps(SIMDE_FLOAT32_C( 0.02425));
+    const simde__m128 high = simde_mm_set1_ps(SIMDE_FLOAT32_C( 0.97575));
+
+    simde__m128 q, r;
+
+    simde__m128 matched;
+
+    /* if (a < 0 || a > 1) */
+    simde__m128 m1 = simde_mm_or_ps(simde_mm_cmplt_ps(a, simde_mm_set1_ps(SIMDE_FLOAT32_C(0.0))), simde_mm_cmpnlt_ps(a, simde_mm_set1_ps(SIMDE_FLOAT32_C(1.0))));
```

Comment:
You should use `simde_mm_cmpgt_ps` instead of `simde_mm_cmpnlt_ps`.  The latter would be the equivalent of `>=` not `>` (though you could also use `simde_mm_cmpge`).